recipes/avahi/all/conanfile.py

```diff
@@ -90,11 +90,13 @@ def package(self):
         tools.rmdir(os.path.join(self.package_folder, "share"))
 
     def package_info(self):
+        self.cpp_info.set_property("cmake_file_name", "Avahi")
         self.cpp_info.names["cmake_find_package"] = "Avahi"
         self.cpp_info.names["cmake_find_package_multi"] = "Avahi"
```

Comment:
where does it come from?

Reply:
I have no idea. I don't think Avahi actually provides anything CMake-related in this regard, right? I was just trying to change something to use `CMakeDeps` and it broke because `find_package()` started failing on `Avahi`. I can adjust accordingly if you let me know what's needed.

Reply:
The PkgConfig targets use the prefix `avahi-`. I'm not sure what the protocol is on Conan Center for providing CMake targets in such a case.

Reply:
it's to not hardcode anything and leave generators with default values (basically <recipeName>).

Reply:
So, close this PR?

Reply:
Inside ConanCenter, `avahi` is only a requirement of `nmos-cpp`. I would agree on removing the `Avahi` renamed target if it is not provided by the library author themselves. At some point in time we will remove it and better sooner than later, don't you think so?

Reply:
no, if we remove these targets, it will break compatibility. there might be consumers outside of the conan-center (e.g. in private repos).

Reply:
I know, but we can claim it's been a bug that we are fixing... and those using rrevs+lockfiles shouldn't be affected.

Reply:
so, do nothing and close PR?

Reply:
Yes, at least we don't want to introduce the _bug_ to other generators (`CMakeDeps`).

---

> Where does it come from? I think these names should never have been added in the first place.

And in a different PR (or changing this one), I would vote for this change.

Reply:
Alright then, I'll close this PR. Thanks all!